packages/trace-viewer/src/sw/traceModelBackends.ts

```diff
@@ -129,14 +128,22 @@ export class FetchTraceModelBackend implements TraceModelBackend {
     const fileName = entries.get(entryName);
     if (!fileName)
       return;
-    return fetch('/trace/file?path=' + encodeURIComponent(fileName));
+
+    return fetch(formatUrl(fileName));
   }
 }
 
+const baseURL = new URL(self.location.href);
+baseURL.port = baseURL.searchParams.get('testServerPort') ?? baseURL.port;
+
 function formatUrl(trace: string) {
```

Comment:
nit: I'd rename to e.g. `formatTraceFileUrl`

Reply:
Good idea!